django/db/backends/__init__.py

```diff
@@ -1232,6 +1235,12 @@ def value_to_db_decimal(self, value, max_digits, decimal_places):
         """
         return utils.format_number(value, max_digits, decimal_places)
 
+    def value_to_db_ipaddress(self, value):
+        """Transform a string representation of an ip address into the expected
```

Comment:
could you use the same doc string style as above?
```
"""
...
"""
```
Also should be "Transforms... " to match coding style guidelines.